JumpRoyale/src/GameObjects/Brick.cs

```diff
@@ -0,0 +1,13 @@
+public record Brick : BaseObject
```

Comment:
The classes that derive from `BaseObject` don't need to exist since they don't override any functionality or provide new functionality. I would delete them all and change `JumpRoyale/src/GameObjects/GameObject.cs` to look like this:

```cs
public static class GameObject
{
    private static readonly Dictionary<GameObjectType, BaseObject> _gameObjects =
        new()
        {
            [GameObjectType.Stone] = new BaseObject(
	            GameObjectAtlasCoords.StoneBlock,
	            GameObjectAtlasCoords.StonePlatformLeft,
	            GameObjectAtlasCoords.StonePlatformMiddle,
	            GameObjectAtlasCoords.StonePlatformRight
	    ),
	    // ...and so on
        };

    public static BaseObject Get(GameObjectType type) => _gameObjects[type];
}
```

Reply:
Changed 👍  less clutter now. In the beginning I expected to use the classes later, but I already forgot what for 😁  it turned out they weren't needed anyway